test/TestCliRolePaths.py

```diff
@@ -82,3 +83,24 @@ def test_run_role_name_invalid(self):
 
         result = run_ansible_lint(cwd=cwd, role_path=role_path)
         assert '106 Role name invalid-name does not match' in str(result)
+
+
+def test_run_playbook_github():
+    """Call ansible-lint simulating GitHub Actions environment."""
+    cwd = str(Path(__file__).parent.parent.resolve())
+    role_path = 'examples/example.yml'
+
+    env = {
+        "GITHUB_ACTIONS": "true",
+        "GITHUB_WORKFLOW": "foo"
+    }
+
+    result_gh = run_ansible_lint(cwd=cwd, role_path=role_path, env=env)
+    result_no_gh = run_ansible_lint(cwd=cwd, role_path=role_path)
+
+    expected = (
+        '::error file=examples/example.yml,line=47::[E101] '
+        'Deprecated always_run'
+    )
+    assert expected in str(result_gh)
+    assert expected not in str(result_no_gh)
```

Comment:
This looks like two distinct tests, they should show up in the report separately. You could use parametrize here.